config/gta3/cleo.xml

```diff
@@ -0,0 +1,958 @@
+      </Args>
+    </Command>
+    <Command ID="0x4c4" Name="GET_OFFSET_FROM_CHAR_IN_WORLD_COORDS" Hash="0x9e04f234">
+      <Args>
+        <Arg Type="INT" Entity="CHAR"/>
+        <Arg Type="FLOAT" Desc="X offset"/>
+        <Arg Type="FLOAT" Desc="Y offset"/>
+        <Arg Type="FLOAT" Desc="Z offset"/>
+        <Arg Type="FLOAT" Desc="X Coord" Out="true"/>
+        <Arg Type="FLOAT" Desc="Y Coord" Out="true"/>
+        <Arg Type="FLOAT" Desc="Z Coord" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x46f" Name="GET_CURRENT_PLAYER_WEAPON" Hash="0xddb22052">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x4dd" Name="GET_CHAR_ARMOUR" Hash="0x2559185e">
+      <Args>
+        <Arg Type="INT" Entity="CHAR"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x4c9" Name="IS_PLAYER_IN_FLYING_VEHICLE" Hash="0xb92a9496">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x4a8" Name="IS_PLAYER_IN_ANY_BOAT" Hash="0xe96e0fea">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x4aa" Name="IS_PLAYER_IN_ANY_HELI" Hash="0x8cc3fb48">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x47e" Name="IS_PLAYER_ON_ANY_BIKE" Hash="0x36f8a3e4">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x485" Name="IS_PC_VERSION" Hash="0x5f927dd1"/>
+    <Command ID="0x59a" Name="IS_AUSTRALIAN_GAME" Hash="0x40172dc1"/>
+    <Command ID="0x601" Name="IS_KEY_PRESSED_WITH_SENSITIVITY" Hash="0x662d65d3">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0x602" Name="EMULATE_KEY_PRESS_WITH_SENSITIVITY" Hash="0x25eda09c">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0x603" Name="IS_CAMERA_IN_WIDESCREEN_MODE" Hash="0xcd6ad16b"/>
+    <Command ID="0x604" Name="GET_MODEL_ID_OF_A_WEAPON" Hash="0x60774d21">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x605" Name="GET_WEAPON_ID_OF_A_MODEL" Hash="0x887e993a">
+      <Args>
+        <Arg Type="INT" Enum="DEFAULTMODEL"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x606" Name="SET_MEMORY_OFFSET" Hash="0x7cab85ce">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0x607" Name="GET_CURRENT_WEATHER" Hash="0xcf0b6bc6">
+      <Args>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x608" Name="DISPLAY_TEXT_WITH_STRING" Hash="0x51e4cae3">
```

Comment:
Let's call it DISPLAY_TEXT_STRING, given it takes a string instead of text label, as opposed to the DISPLAY_TEXT_WITH_* commands.